examples/python/circuit_draw.py

```diff
@@ -29,5 +29,8 @@ def build_bell_circuit():
 # Create the circuit
 bell_circuit = build_bell_circuit()
 
-# Use the internal .draw() to print the circuit
+# Print the circuit directly to stdout
 print(bell_circuit)
+
+# Or use the internal .draw() to print the circuit
+bell_circuit.draw()
```

Comment:
Since this is a script it won't print anything (unless you copy it over e.g. to a Jupyter notebook cell). A better solution might just be to change the comment on top of `print(circuit)` to just say something like "Print the circuit, which implicitly calls bell_circuit.draw(output="text")" or something?

Reply:
yeah that's even better! thx